db_migrator--1.1.0.sql

```diff
@@ -2405,3 +2098,48 @@ END;$$;
 
 COMMENT ON FUNCTION db_migrate(name,name,name,name,name[],jsonb,boolean) IS
    'migrate a remote database from a foreign server to PostgreSQL';
+
+CREATE FUNCTION execute_statement(
```

Comment:
Perhaps you could use a name that is less generic, like `db_migrate_execute_statement`.
But I have no strong opinion about that.

All other functions have a comment. It would be good to have a comment to this function as well.

Since this is a function that the end user may call, it should be documented.

Reply:
I know what you mean. From my understanding, `db_migrate` prefix was used by high-level (or migration) functions like `db_migrate()` or `db_migrate_indexes()`.

Do you want a global prefix for all functions? It seems that this is not yet the rule:

```text
contrib_regression=# \dx+ db_migrator 
              Objects in extension "db_migrator"
                      Object description                       
---------------------------------------------------------------
 function adjust_to_bigint(numeric)
 function construct_indexes_statements(name,name)
 function db_migrate_constraints(name,name)
 function db_migrate_finish(name,name)
 function db_migrate_functions(name,name)
 function db_migrate_indexes(name,name)
 function db_migrate_mkforeign(name,name,name,name,jsonb)
 function db_migrate(name,name,name,name,name[],jsonb,boolean)
 function db_migrate_prepare(name,name,name,name,name[],jsonb)
 function db_migrate_refresh(name,name,name,name[])
 function db_migrate_tables(name,name,boolean)
 function db_migrate_triggers(name,name)
 function db_migrate_views(name,name)
 function execute_statement(text,name,name,text,name)
 function materialize_foreign_table(name,name,boolean,name)
```

Reply:
That's cool, jet's just leave the name as it is then.